test/CMakeLists.txt

```diff
@@ -57,15 +61,16 @@ endif()
 
 enable_testing()
 
-# Note: doctest and similar testing frameworks can automatically configure CMake tests For other
-# testing frameworks add the tests target instead: ADD_TEST(GreeterTests GreeterTests)
+# Note: doctest and similar testing frameworks can automatically configure CMake tests.
+# include(${doctest_SOURCE_DIR}/scripts/cmake/doctest.cmake)
+# doctest_discover_tests(GreeterTests)
 
-include(${doctest_SOURCE_DIR}/scripts/cmake/doctest.cmake)
-doctest_discover_tests(GreeterTests)
+# For other testing frameworks add the tests target instead:
+ADD_TEST(GreeterTests GreeterTests)
```

Comment:
`doctest_discover_tests` is a great feature, why remove it?

Reply:
If doctest is installed on build host and used, this does not work!